Lib/csv.py

```diff
@@ -416,6 +419,10 @@ def has_header(self, sample):
                 continue # skip rows that have irregular number of columns
 
             for col in list(columnTypes.keys()):
```

Comment:
Wonder if this is a 2to3 conversion idiom that’s not truly necessary (`for col in columnTypes`)